integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/EnterAmountToTransferFragment.kt

```diff
@@ -228,4 +227,10 @@ class EnterAmountToTransferFragment : Fragment(R.layout.enter_amount_to_transfer
             }
         }
     }
+
+    fun showKeyboardAndButton(isVisible: Boolean) {
+        if (isAdded) { // avoid IllegalStateException
+            binding.keyboardContainer.isVisible = isVisible
```

Comment:
For some reason this doesn't work after leaving this screen and returning to it.

```
         viewModel.keyboardStateCallback.value = isVisible
         ```